tests/migration/postcopy.go

```diff
@@ -168,39 +168,32 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 		)
 
 		Context("and fail", Serial, func() {
-			var createdPods []string
-			BeforeEach(func() {
-				createdPods = []string{}
-			})
+			var killerPod string
 
-			runMigrationKillerPod := func(nodeName string) {
-				podName := fmt.Sprintf("migration-killer-pod-%s", rand.String(5))
+			runVirtHandlerKillerPod := func(nodeName string) {
+				podName := "migration-killer-pod-"
```

Comment:
nit: maybe we want to call it (and the corresponding argument to `RenderPrivilegedPod()`) something like "name pattern"?